src/appleseed.studio/mainwindow/droppedmaterialmenu.cpp

```diff
@@ -0,0 +1,74 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Thibault Vergne, Jupiter Jazz Limited
```

Comment:
The copyright notice for all files created this year should simply be

    Copyright (c) 2019 Thibault Vergne, The appleseedhq Organization

More details here: https://github.com/appleseedhq/appleseed/wiki/Copyright-Notices